vera/README.md

```
@@ -93,7 +93,7 @@ execute(compile_result, ...)    # → run WASM via wasmtime
 | `verifier.py` | 8,190 | Verify | Contract verification | `verify()` |
 | `wasm/` | 25,941 | Compile | WASM translation layer (package) | `WasmContext`, `WasmSlotEnv`, `StringPool` |
 | ` ├ context.py` | 1,089 | | Composed WasmContext, expression dispatcher, block translation | |
-| ` ├ helpers.py` | 463 | | WasmSlotEnv, StringPool, type mapping, array element helpers | |
+| ` ├ helpers.py` | 541 | | WasmSlotEnv, StateClauseEntry, StringPool, type mapping, array element helpers | |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)